src/components/menus/Items.ts

```diff
@@ -114,6 +115,29 @@ export class Items<TGameStartr extends FullScreenPokemon> extends Component<TGam
         const options: any[] = [
             {
                 callback: (): void => {
+                    const category = this.gameStarter.itemsHolder.getItem(listing.item).category;
```

Comment:
I added a `features` component that has a bool getter for whether this is allowed. You should only add this object to the `options` array if it is.

You'll need to merge the upstream master into your branch.